setup-registry/action.yaml

```diff
@@ -23,7 +27,14 @@ runs:
       uses: imjasonh/setup-crane@v0.3
       with:
         version: ${{inputs.crane-version}}
+
+    - name: Run registry
+      if: ${{inputs.disk}} == 'true' && ${{inputs.crane-version}} == 'tip'
+      shell: bash
+      run: |
+        PORT=${{inputs.port}} crane registry serve --blobs-to-disk=true &
```

Comment:
So I have concerns...

Once we snap a release, how are we going to support released versions in this?

If we kill `crane-version` and instead pass `tip` to `setup-crane` when `inputs.disk` is true, then I think we have recourse to then switch to `--blobs-to-disk=${{ inputs.disk }}` after a crane release, right?

Reply:
We should also add test coverage for the disk path to make sure the actions are properly predicated.

Reply:
@imjasonh I think you did the opposite of what I was thinking 🤣